### PR TITLE
issue/2799 Removed redundant behaviour

### PR DESCRIPTION
**Pull request by oliverfoster**
requires https://github.com/adaptlearning/adapt-contrib-core/pull/84
part of https://github.com/adaptlearning/adapt_framework/issues/2799

### Changed
* Switched to components.register from Adapt.register
* Use `a11y`, `device` and `notify` directly

### CODE REVIEW

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

File: js/NarrativeView.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
 import Adapt from 'core/js/adapt';
+import components from 'core/js/components';
 import a11y from 'core/js/a11y';
+import device from 'core/js/device';
+import notify from 'core/js/notify';
 import ComponentView from 'core/js/views/componentView';
 import device from 'core/js/device';
```
Comment (joe-allen-89) [inline js/NarrativeView.js:7]: Need to remove the duplicate import device.

**Review by joe-allen-89 — CHANGES_REQUESTED** [review]
Duplicate import device.

**Review by eleanor-heath — APPROVED** [review]

File: js/NarrativeView.js (hunk 1 above)
Comment (oliverfoster) [reply js/NarrativeView.js]: Weird that I keep doing that.

**Pull request merged**